pages/pipelines/incoming_webhooks.md

```diff
@@ -10,23 +10,22 @@ Buildkite only logs and temporarily stores the incoming webhook information as i
 Buildkite stores the information on recent incoming webhooks received per pipeline for approximately one week. It is used for troubleshooting purposes only — for example, to diagnose why a given hook didn't trigger a build. This functionality is not exposed in the UI.
 
 ## What security information is stored in the secret in a Buildkite URL?
-Secrets are pipeline-specific. A secret can't prove that it hasn't been acquired by somebody else and added as a webhook inside their own GitHub repository (which is equally true of any other secret). However, all incoming webhooks coming from GitHub-hosted repositories are IP-filtered to provide an additional layer of security. As a Buildkite user, you can [set up a pipeline and request a URL](/docs/integrations/github#set-up-a-new-pipeline-for-a-github-repository) to add it to your GitHub organization as a webhook. It means that you would prove that it is coming from your GitHub organization.
+Secrets are pipeline-specific. A secret can't prove that it hasn't been acquired by somebody else and added as a webhook inside their own GitHub repository (which is equally true of any other secret). However, all incoming webhooks from GitHub-hosted repositories are IP-filtered to provide an additional layer of security. As a Buildkite user, you can [set up a pipeline and request a URL](/docs/integrations/github#set-up-a-new-pipeline-for-a-github-repository) to add it to your GitHub organization as a webhook, proving it comes from your GitHub organization.
 
-IP filtering is currently only available for:
+IP filtering is available for:
 
 * GitHub
 * GitHub Enterprise Server
-* Bitbucket Server
+* Bitbucket Server (not Bitbucket Cloud)
 * GitLab Community
 * GitLab Enterprise
 
-To enable IP filtering for these repository providers:
+For cloud products like GitHub, Buildkite manages a set of well-known IP addresses and filters them automatically. For self-hosted solutions like GitHub Enterprise Server, you can provide the IP addresses to filter.
```

Comment:
This is the real change. The rest are just style nitpicks while I'm in this section.

Reply:
Do we maybe want to add a link to the well known IPs here?

Reply:
👍🏻 Added in [d777661](https://github.com/buildkite/docs/pull/2338/commits/d7776619314558b7f8e742088b319a7838126916)